packages/mui-material/src/Alert/Alert.js

```diff
@@ -356,16 +373,48 @@ Alert.propTypes /* remove-proptypes */ = {
    * @default {}
    */
   slotProps: PropTypes.shape({
```

Comment:
Shouldn't `slotProps` have a similar structure to:

https://github.com/mui/material-ui/blob/03325351a4c3a7b1ef9adbb7ad49a5caa39559a9/packages/mui-material/src/Tooltip/Tooltip.js#L954-L959

I think all (or at least most) `slotProps`'s prop types follow this. For example, the `closeButton` and `closeIcon` in this same type.

Reply:
It's generated from `pnpm proptypes`, I'm figuring out why it does not follow the same structure as the `closeButton`.

Reply:
This is fixed after applying the suggestion from @sai6855